egui-winit/Cargo.toml

```diff
@@ -22,7 +22,7 @@ all-features = true
 
 [dependencies]
 egui = { version = "0.15.0", path = "../egui", default-features = false, features = ["single_threaded"] }
-winit = "0.25"
```

Comment:
We should mention the winit version bump in `egui-winit/CHANGELOG.md`

Reply:
Updated changelogs!